specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -1098,6 +1098,232 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/getReservationDetailsReport": {
+      "post": {
+        "tags": [
+          "ReservedInstances"
+        ],
+        "operationId": "GetReservationDetailsReport_ByBillingAccountId",
+        "description": "Lists the reservations details for provided date range asynchronously based on enrollment id.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "ReservationDetails": {
+            "$ref": "./examples/GetReservationDetailsReportByBillingAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/enrollmentIdParameter"
+          },
+          {
+            "$ref": "#/parameters/startDateParameter"
+          },
+          {
+            "$ref": "#/parameters/endDateParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Request will be processed. Use the Operation-Location header to check the status.",
+            "headers": {
+              "Operation-Location": {
```

Comment:
this is not an appropriate header.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#call-action-post-asynchronously

Reply:
will change to "Location" header